style.css

```diff
@@ -19,3 +19,21 @@ body {
   border-top: 1px solid #007bff;
   border-bottom: 3px solid orangered;
 }
+
+.switch-button {
```

Comment:
this class should be given a `margin-block`

Reply:
What should be the size of the margin?

Reply:
Up to you, just mentioning it as it appears squished into a gap as it is